spec/models/az322_contribution_spec.rb

```diff
@@ -99,6 +99,30 @@
         az.valid?
         expect(az.errors[:amount]).to be_empty
       end
+
+      it "should validate format" do
+        az.amount = 10.1
+        az.valid?
+        expect(az.errors[:amount]).to be_empty
+
+        az.amount = 10.22
+        az.valid?
+        expect(az.errors[:amount]).to be_empty
+
+        az.amount = '.1'
+        az.valid?
+        expect(az.errors[:amount]).to be_empty
+
+        az.amount = '.12'
+        az.valid?
+        expect(az.errors[:amount]).to be_empty
+      end
+
+      it "should clean up format that leads to validation errors" do
+        az.amount = '1.'
+        az.valid?
+        expect(az.errors[:amount]).to be_empty
```

Comment:
should it also test what it changes the value to? (in addition to just not being invalid)

Reply:
done ❇️  https://github.com/codeforamerica/vita-min/pull/4770/commits/bd9ec35e1eadda8e63f5712d492bc5370e3a06a3